pkg/controller/persistentvolume/types.go

```diff
@@ -26,6 +26,14 @@ import (
 )
 
 const (
+	// A PVClaim can request a quality of service tier by adding this annotation.  The value of the annotation
+	// is arbitrary.  The values are pre-defined by a cluster admin and known to users when requesting a QoS.
+	// For example tiers might be gold, silver, and tin and the admin configures what that means for each volume plugin that can provision a volume.
+	qosProvisioningKey = "volume.experimental.kubernetes.io/quality-of-service"
```

Comment:
wouldn't the standard be something like `kubernetes.io/volumes.experimental.qos`.  Also, "quality of service" seems to imply something but this is really just a label so `kubernetes.io.volumes.experiemental.storage-class` seems to make sense and corresponds with the cli flag. Unless, of course, this name has already been agreed upon elsewhere.